lm-composed/etc/seed-license-in-slurm.py

```diff
@@ -0,0 +1,43 @@
+#!/app/lm-agent/.venv/bin/python
+import subprocess
+
+
+LICENSE_NAME = "test_product.test_feature"
+SERVER_TYPE = "flexlm"
+
+
+def main():
+    output = subprocess.run([
+        "scontrol",
+        "show",
+        "lic",
+        f"{LICENSE_NAME}@{SERVER_TYPE}",
+    ], capture_output=True)
+
+    stdout = output.stdout.decode("utf-8")
+
+    if "test_product.test_feature@flexlm" in stdout:
+        return
+    else:
+        output = subprocess.run([
+            "sacctmgr",
```

Comment:
See my shlex comment above. It really helps readability.

Reply:
This one was a bit weird with `shlex`, but I was able to break it down to 2 lines.
What do you think?

Reply:
It looks ok, but you will need to add an extra space at the beginning of the second line.